pallets/elections/src/lib.rs

```diff
@@ -271,6 +271,13 @@ pub mod pallet {
         NonUniqueListOfValidators,
     }
 
+    impl<T: Config> OnValidatorDisabled for Pallet<T> {
+        fn disable_validator(_validator_index: u32) {
```

Comment:
Is this unimplemented on purpose?

Reply:
yes yes, this is only for presentation and I wasn't sure how proper handling should look like here :man_shrugging: